static/modules/gamemanager.js

```diff
@@ -406,6 +446,10 @@ class GameManager {
       deltaTime = 200;
     }
     this.totalTime += deltaTime;
+
+    // Game time doesn't incrememnt when the game is paused.
+    if (!this.gamePause) this.gameTime += deltaTime;
```

Comment:
This is probably the right call for now. But now I'm thinking about if we want smoothly animated elements on the pause screen or something.

Reply:
Then we use getTotalTime(). That's the difference between the two numbers.